README.md

```diff
@@ -21,65 +16,65 @@ It is a tiny alternative to more capable projects:
 - Representing state in a map:
   [robertluo/fun-map](https://github.com/robertluo/fun-map)
 
-## TL;DR example
-
 ``` clojure
-;; in your project
-(require '[piotr-yuxuan/closeable-map :refer [closeable-map]])
+;; In your project, require:
+(require '[piotr-yuxuan.closeable-map :refer [closeable-map closeable-hash-map] :as system])
```

Comment:
Actually it's not possible to have
``` clj
(require '[piotr-yuxuan.closeable-map :refer [closeable-map closeable-hash-map] :as closeable-map])
```
as it would be ambiguous for the compiler. I think that we can find a better alias that `system`. Otherwise, we have several options:
- Rename the project and its main ns
- Rename the function `closeable-map`
- Find a better alias.